test/e2e/framework/framework.go

```diff
@@ -208,6 +208,45 @@ func waitForPodsReady(client v1client.CoreV1Interface, timeout time.Duration, ex
 	}
 }
 
+func (f *Framework) WaitForServiceReady(timeout time.Duration, serviceName string) error {
```

Comment:
@brancz What do you think about a centralized standardized timeout value. Might come in handy when we have to increase it in case of slow infrastructure.